.github/workflows/kurl-addon-kots-test.yaml

```diff
@@ -41,7 +45,7 @@ jobs:
         uses: ./.github/actions/kurl-addon-kots-generate
         with:
           addon_version: "${{ inputs.addon_version }}"
-          s3_prefix: "${{ github.event.action == 'workflow_dispatch' && 'test/' || inputs.s3_prefix }}"
```

Comment:
I'm not 100% sure this change is equivalent since the prefix would need to empty if the event action is not a `workflow_dispatch`, but now `s3_prefix` has a default of `test/`. I'll defer to @emosbaugh since I'm not too familiar with what the prefix value should be.

Reply:
This should be safe. There is no way that expression could have evaluated to anything but an empty string.